### ISSUE TITLE
Media captions support for accessibility

### ISSUE THREAD

**Issue by productboard-ac** [microsoft/AdaptiveCards#4963]
# Description

Reference: https://captionmax.com/common-subtitle-prerecorded-closed-caption-file-types/

Accessibility bug: [28383785](https://microsoft.visualstudio.com/OS/_workitems/edit/28383785)

## Open Issues

- [ ] iOS Sidecar support?
- [ ] Android sidecar support?
- [ ] WPF sidecar support?
- [x] UWP sidecar support?
- [x] JS sidecar support?
- [ ] What about per-language captions?

## Requirements

1. If a video is encoded with closed captions they should be automatically supported on each platform (if the media mimeType is supported)
2. If the video file does not have captions, one or more "sidecar" caption formats can be provided, which will be passed along to the Renderer/Native Platform Media player as appropriate.

### Media

One new property on `Media`

| Property       | Type         | Required | Description |
| -------------- | ------------ | -------- | ----------------- |
| captions | `CaptionSource[]` | No | Array of "sidecar" close caption sources that will be passed to the underlying platform.

### CaptionSource

New type called `CaptionSource`

| Property       | Type         | Required | Description |
| -------------- | ------------ | -------- | ----------------- |
| mimeType | `string` | Yes | Mime type of associated closed captions (e.g. "scc").
| url | `uri` | Yes | URL to closed captions file.
| label | `label` | Yes | Label to display for this caption track.

## Examples

```json
{
	"$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
	"type": "AdaptiveCard",
	"version": "1.1",
	"fallbackText": "This card requires Media to be viewed. Ask your platform to update to Adaptive Cards v1.1 for this and more!",
	"body": [
		{
			"type": "Media",
			"poster": "https://adaptivecards.io/content/poster-video.png",
			"altText": "Adaptive Cards overview video",
			"sources": [
				{
					"mimeType": "video/mp4",
					"url": "https://adaptivecardsblob.blob.core.windows.net/assets/AdaptiveCardsOverviewVideo.mp4"
				}
			],
			"captionSources": [
				{
					"mimeType": "vtt",
					"label": "English (vtt)",
					"url": "https://gist.githubusercontent.com/PankajBhojwani/d11855a1e0c0b1f70a0e64f81a1fb696/raw/46fa93da5c3703d03843d59cedd8b5f10aa3747b/test.vtt"
				}
			]
		}
	],
	"actions": [
		{
			"type": "Action.OpenUrl",
			"title": "Learn more",
			"url": "https://adaptivecards.io"
		}
	]
}
```

**Comment by productboard-ac** [comment]
Link to feature: https://adaptivecards.productboard.com/feature-board/planning/features/5952351

**Comment by jonmill** [comment]
Moved out of Current Sprint

**Comment by beervoley** [comment]
@PankajBhojwani could you please edit the schema here to support **language** as well?:)
